build.gradle.kts

```
@@ -261,6 +263,12 @@ allprojects {
                 }
             }
 
+            apply(plugin = "biz.aQute.bnd.builder")
+            register<Bundle>("bundle") {
+                bundle {
+                }
```

When I added the following, the configuration was just ignored and the generated MANIFEST.MF in `weisJ-jsvg\jsvg\build\libs\jsvg-1.6.2-SNAPSHOT.jar` did had the content that should result from the specified instructions.
```suggestion
                bundle {
                    bnd("""
Bundle-SymbolicName: com.github.weisj.jsvg
-exportcontents: !*.impl.*,*
-removeheaders: Private-Package,Tool
""")
                }
```

---

I don't know why it is ignored. The documentation on this isn't very clear in how you are supposed to set things up. I have moved bundle setup into the gradle file of the jsvg itself (not the root) and now it works. See latest commit for this.

---

> now it works.

That's really awesome. Thank you.

> I don't know why it is ignored. 

I'm sorry, but I cannot answer that since by Gradle knowledge is almost zero and the more I tried to make this change work the more I felt it's not only close to but zero.